### PR TITLE
Add handling of self webworker context

### LINKED ISSUE #47: Won't work in browser's web worker
Here https://github.com/pine/arraybuffer-loader/blob/master/lib/to-array-buffer.js#L4, it checks for `window`, but needs also check for `self` to make it work in a web worker.

### PR DESCRIPTION
**Pull request by wclr**
Fixes #47

Formatted with default prettier settings.

### CODE REVIEW

File: lib/arraybuffer-loader.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
-const loaderUtils = require('loader-utils')
+const loaderUtils = require("loader-utils");
 
 module.exports = function (content) {
-  if (this.cacheable) { this.cacheable() }
+  if (this.cacheable) {
+    this.cacheable();
+  }
 
+  const request = require.resolve("./to-array-buffer.js");
```
Comment (ghost) [inline lib/arraybuffer-loader.js:10]: This repository appears to use single quotes only @wclr

**Review by ghost — CHANGES_REQUESTED** [review]
as I understand it, single quotes please!

File: lib/arraybuffer-loader.js (hunk 1 above)
Comment (pine) [reply lib/arraybuffer-loader.js:10]: Hello @M00TSDNS.

Thank you for your review, and I'm sorry for waiting you.
I will check the Pull Request today.

**Comment by pine** [comment]
Hello @wclr.
Thank you for your contribution. I'm sorry for waiting you long time.

I'm concerned about code style @M00TSDNS also said, but I will fix it after merging this.

**Comment by ghost** [comment]
It's no worries Pine, hope life is treating you well!

**Pull request merged**